zebra-consensus/src/block.rs

```diff
@@ -74,29 +75,26 @@ where
             let hash = block.hash();
 
             // Check that this block is actually a new block.
-            match state_service.ready_and().await?.call(zs::Request::Depth(hash)).await? {
+            match state_service
+                .ready_and()
+                .await?
```

Comment:
These errors should be getting wrapped in our error enums rather than being returned straight away, so we get something like "verification for block {} failed: some zebra-state error message" instead of just "some zebra-state error message"